pinot-common/src/main/java/org/apache/pinot/common/utils/TarGzCompressionUtils.java

```diff
@@ -123,6 +132,16 @@ public static List<File> untar(File inputFile, File outputDir)
    */
   public static List<File> untar(InputStream inputStream, File outputDir)
       throws IOException {
+      return untarRateLimit(inputStream, outputDir, NO_RATE_LIMIT);
+  }
+
+  /**
+   * Un-tars an inputstream of a tar.gz file into a directory, returns all the untarred files/directories.
+   * RateLimit limits the untar rate
+   * <p>For security reason, the untarred files must reside in the output directory.
+   */
+  public static List<File> untarRateLimit(InputStream inputStream, File outputDir, long rateLimit)
```

Comment:
for the rateLimit argument, can we include the units in the argument name to make code more readable?